### PR TITLE
New package: rr-4.0.1

### PR DESCRIPTION
**Pull request by Spencer-H**

### CODE REVIEW

**Comment by Spencer-H** [comment]
#3694

**Comment by Spencer-H** [comment]
This build should be passing. The package is marked to only compile for i686 and x86_64, I have no idea why travis is trying to build the others and failing it.

**Comment by Spencer-H** [comment]
Oh thanks. I'm an idiot some times.

File: common/shlibs [hunk 1]
```diff
@@ -2354,3 +2354,5 @@ libfilezilla.so.0 libfilezilla-0.3.1_1
 libtommath.so.1 libtommath-1.0_1
 libunicorn.so.0 unicorn-0.9_1
 libglyphy.so.0 glyphy-0.0.20160104_1
+librrpreload_32.so rr-4.0.1_1
+librrpreload.so rr-4.0.1_1
```
Comment (xtraeme) [inline common/shlibs]: These changes to `common/shlibs` are unnecessary, those shared libraries are not meant to be used by other applications.

**Comment by Spencer-H** [comment]
Alright. Fixed.

**Pull request merged**